pkg/operator/ceph/webhook-config.go

```diff
@@ -195,9 +182,25 @@ func createValidatingWebhookConfig(ctx context.Context, clusterdContext *cluster
 		validatingWebhook.Webhooks = append(validatingWebhook.Webhooks, webhookRules(resourcesWebhookName[i], resources[i], resourcesServicePath[i]))
 	}
 
-	_, err := clusterdContext.Clientset.AdmissionregistrationV1().ValidatingWebhookConfigurations().Create(ctx, validatingWebhook, metav1.CreateOptions{})
+	webhookConfig, err := clusterdContext.Clientset.AdmissionregistrationV1().ValidatingWebhookConfigurations().Get(ctx, validatingWebhook.Name, metav1.GetOptions{})
+	if err != nil {
+		if apierrors.IsNotFound(err) {
+			_, err := clusterdContext.Clientset.AdmissionregistrationV1().ValidatingWebhookConfigurations().Create(ctx, validatingWebhook, metav1.CreateOptions{})
+			if err != nil {
+				logger.Errorf("failed to create admission webhook %s. %v", webhookConfigName, err)
```

Comment:
we can remove this line since we are returning same in next line

Reply:
This is for logging, we had it earlier also. it helps in debugging.